.github/workflows/build.yml

```diff
@@ -46,40 +69,179 @@ jobs:
         run: pre-commit install-hooks
       - name: Run pre-commit on all files
         run: pre-commit run --all-files
-  build:
+
+  prepare:
+    # Calculates and publishes outputs that are used by other jobs.
+    #
+    # Outputs:
+    #   created:
+    #     The current date-time in RFC3339 format.
+    #   repometa:
+    #     The json metadata describing this repository.
+    #   source_version:
+    #     The source version as reported by the `bump_version.sh show` command.
+    #   tags:
+    #     A comma separated list of Docker tags to be applied to the images on
+    #     DockerHub.  The tags will vary depending on:
+    #     - The event that triggered the build.
+    #     - The branch the build is based upon.
+    #     - The git tag the build is based upon.
+    #
+    #     When a build is based on a git tag of the form `v*.*.*` the image will
+    #     be tagged on DockerHub with multiple levels of version specificity.
+    #     For example, a git tag of `v1.2.3+a` will generate Docker tags of
+    #     `:1.2.3_a`, `:1.2.3`, `:1.2`, `:1`, and `:latest`.
+    #
+    #     Builds targeting the default branch will be tagged with `:edge`.
+    #
+    #     Builds from other branches will be tagged with the branch name. Solidi
```

Comment:
I'm going to insist that we change `Solidi` to `Solidi ("slash")`, and I suggest that we provide a link for solidus here.  This is some pretty arcane terminology, which I love, but we need to offer the reader
* The vernacular word, so he or she is not lost
* A link to wikipedia or whatever for his or her edification

Perhaps our readers will one day become so numerous that "solidus" becomes a common word (like "octothorpe") and we can remove the parenthesized vernacular.

Reply:
**Solidus**, the guitarist from GNR
![image](https://user-images.githubusercontent.com/5521725/98273602-737faa00-1f60-11eb-98e4-0aa2e220a582.png)

Reply:
[This](https://en.wikipedia.org/wiki/Slash_(punctuation)#Encoding) looks to be an excellent link to use.